backend/Application/DTOs/Users/CreateUser/CreateUserRequest.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Domain.Shared.Enums;
+using System.ComponentModel.DataAnnotations;
+
+namespace Application.DTOs.Users.CreateUser;
+
+public class CreateUserRequest
+{
+    [Required]
+    public string FirstName { get; set; } = null!;
+
+    [Required]
+    public string LastName { get; set; } = null!;
+
+    [Required]
+    public DateTime DateOfBirth { get; set; }
+
+    [Required]
+    public Genders Gender { get; set; }
+
+    [Required]
+    public DateTime JoinedDate { get; set; }
+
+    [Required]
+    public UserRoles Role { get; set; }
+
+    [Required]
+    public Locations Location { get; set; }
```

Comment:
Enum convension ✔️ DO use a singular type name for an enumeration unless its values are bit fields.

https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces#naming-enumerations
UserRoles, Locations, Genders must be : UserRole,Location,Gender